Changes for successful compilation on Solaris.

Evaluate to possibly send upstream

--- rtorrent-0.8.2/src/rpc/parse.cc	2016-07-21 14:56:37.128876142 -0700
+++ rtorrent-0.8.2/src/rpc/parse.cc	2016-07-21 14:56:47.788833146 -0700
@@ -37,6 +37,7 @@
 #include "config.h"
 
 #include <locale>
+#include <cstring>
 #include <rak/algorithm.h>
 #include <rak/path.h>
 #include <torrent/exceptions.h>
